gems/pending/metadata/linux/LinuxSystemd.rb

```diff
@@ -41,6 +42,10 @@ def files(unit_extension)
       end
     end
 
+    def existing_dirs
+      (ALL_DIRS).select  { |dir| @fs.fileExists?(dir) }
```

Comment:
Same about the parens...also you have an extra unneeded space after the `select` (very minor)